object_storage_api/.env.example

```
@@ -20,4 +20,5 @@ OBJECT_STORAGE__SECRET_ACCESS_KEY=example_password
 OBJECT_STORAGE__BUCKET_NAME=object-storage
 OBJECT_STORAGE__PRESIGNED_URL_EXPIRY_SECONDS=1800
 ATTACHMENT__MAX_SIZE_BYTES=104857600
+IMAGE__ALLOWED_FILE_EXTENSIONS=[".jpg", ".jpeg", ".png", ".bmp", ".tiff", ".webp"]
```

```suggestion
IMAGE__ALLOWED_FILE_EXTENSIONS=[".jpg", ".jpeg", ".jpe",".png", ".bmp", ".tiff", ".tif", ".webp"]
```
There are few more valid extensions for some of these content types. The mimetypes library has a content-type to file extension mapping that you can access to see all of the valid file extensions they cover (I've added all of the missing ones in my suggestion).

You can see the dictionary yourself with the code below.
```
mimetype = mimetypes.MimeTypes()
print(mimetype.types_map_inv)
```

---

Thanks for that!